### PR TITLE
Jacdac gallery

### PR DESCRIPTION
**Pull request by pelikhan**
This is a first revision on the jacdac gallery. It is currently linking to external content. Built-in tutorials + youtube videos are planned and will be added in the future.
image

### CODE REVIEW

**Comment by abchatra** [comment]
How is the teacher\student going to use this? I am not sure we should advertise unless there is hardware to buy?

**Comment by pelikhan** [comment]
Hardware is available (https://www.kittenbot.cc/collections/frontpage/products/kittenbot-jacdac-kit-for-micro-bit) and linked from all pages.

**Comment by abchatra** [comment]
Oh cool. I missed the news!

**Review by ganicke — APPROVED** [review]
Wow, twisted sounds and stuff.

File: docs/jacdac.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+Connect and Code. Instantly.
+
+## Getting started
+
+[Jacdac](https://aka.ms/jacdac) is a plug-and-play hardware accessory system that provides simulation and physical device twin in MakeCode.
+
+```codecard
+[{
+  "name": "Getting started",
+  "description": "Connect and Code. Instantly in MakeCode.",
```
Comment (ganicke) [inline docs/jacdac.md:12]: Connect it, code it. It just works instantly.

Comment (pelikhan) [reply docs/jacdac.md:12]: it is kind of our glorious tag line...
image

**Comment by pelikhan** [comment]
@abchatra here's the long version of the announce https://microsoft.github.io/jacdac-docs/blog/jacdac-for-makecode-and-microbit/ . Planning on adding youtube videos for each activities.

**Pull request merged**